Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Build App
-FROM openjdk:8-jdk-alpine AS BUILD
+FROM openjdk:11-jdk-slim AS BUILD
 
 ADD . /work
 WORKDIR /work
 RUN ./gradlew build --build-file ./build.gradle
 
 # Run App
-FROM openjdk:8-jre-alpine
+FROM openjdk:11-jre-slim
 
 COPY --from=BUILD /work/build/libs/httpstub.jar /app/httpstub.jar
 EXPOSE 8080
 WORKDIR /app
-ENTRYPOINT /usr/bin/java -jar httpstub.jar
\ No newline at end of file
+ENTRYPOINT java -jar httpstub.jar
```

Comment:
Use an array syntax here like `ENTRYPOINT ["java", "-jar"]`. See the section 4 of http://crosbymichael.com/dockerfile-best-practices.html.